_en/2015-08-11-about.html

```diff
@@ -14,24 +14,27 @@
             Vi har avdelinger over hele landet – Stavanger, Bergen, Trondheim, Vestfold, Sørlandet og Oslo. Hver eneste måned skjer
             det kule ting i javaBin-regi – både meetups, studentarrangementer, arrangementer for barn, og mye annet spennende!-->
 
-            javaBin is one of the largest community groups in Norway. It is driven by volunteer enthusiasts from across the country.
-            We have branches throughout the country - Stavanger, Bergen, Trondheim, Vestfold, Southern Norway and Oslo. We organize events
-            almost every month in all javaBin branches - both meetups, student events, events for kids, and much more exciting stuff!
+            javaBin is one of the largest communities in Norway. It is driven by volunteer enthusiasts from across the country. 
+            We have branches throughout the country – in Stavanger, Bergen, Trondheim, Vestfold, Southern Norway and Oslo. 
+            We organize events almost every month in all branches of javaBin. It is both meetups, events for students, 
+            events for kids, and other exciting stuff!
         </p>
         <p>
             <!-- Hvert år i september er det duket for javaBins store happening – <a href="http://javazone.no">JavaZone</a>.
             Det stemmer! En av verdens største Java-konferanser er drevet av en frivillig organisasjon! 3.000 deltakere,
             200 foredragsholdere, 45 partnerbedrifter og generelt sett god stemning! -->
 
-            That's right! One of the world's biggest Java conferences is run by an non-profit organization! 3,000 participants,
-            200 lecturers, 45 partner companies and general good mood!
+            Our biggest happening of the year – <a href="http://javazone.no">JavaZone</a> – takes place every September.
+            That's right! One of the world's biggest Java conferences is run by a non-profit organization! 3,000 participants,
+            200 speakers, 50+ partners, and generally lots of knowledge sharing and fun!
         </p>
         <p>
             <!--Bli med på et javaBin-arrangement da vel! Og er du ekstra interessert, kan du snakke med oss og bli en av arrangørene.
             Vi er over 50 mennesker som bidrar med å skape kule arrangementer for våre nesten 3.000 medlemmer!-->
 
-            Join a javaBin event! And if you are really interested, you can talk with us and become one of the organizers.
-            We have over 50 people who contribute to create cool events for our nearly 3,000 members!
+            We would like you to join one of our events! Also, if you are interested, you ask us about becoming one of the organizers.
+            We have over 50 people who contribute to the community, creating events for our nearly 3,000 members! And we always looking
+            for new friends!
```

Comment:
So what are you waiting for? Come and join one of our events! And if you want to give something back then just ask about becoming an organizer, or how you can contribute in other ways. We currently have over 50 people who contribute, creating events for our nearly 3,000 members! We are always looking for new friends.